tests/bsim/bluetooth/audio/src/cap_initiator_unicast_test.c

```diff
@@ -248,42 +356,111 @@ static void init(void)
 	}
 }
 
+static void cap_device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
+			     struct net_buf_simple *ad)
+{
+	char addr_str[BT_ADDR_LE_STR_LEN];
+	struct bt_conn *conn;
+	int err;
+
+	/* We're only interested in connectable events */
+	if (type != BT_HCI_ADV_IND && type != BT_HCI_ADV_DIRECT_IND) {
+		return;
+	}
+
+	conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, addr);
+	if (conn != NULL) {
+		/* Already connected to this device */
+		bt_conn_unref(conn);
+		return;
+	}
+
+	bt_addr_le_to_str(addr, addr_str, sizeof(addr_str));
+	printk("Device found: %s (RSSI %d)\n", addr_str, rssi);
+
+	/* connect only to devices in close proximity */
+	if (rssi < -70) {
+		FAIL("RSSI too low");
+		return;
+	}
+
+	printk("Stopping scan\n");
+	if (bt_le_scan_stop()) {
+		FAIL("Could not stop scan");
+		return;
+	}
+
+	err = bt_conn_le_create(
+		addr, BT_CONN_LE_CREATE_CONN,
+		BT_LE_CONN_PARAM(BT_GAP_INIT_CONN_INT_MIN, BT_GAP_INIT_CONN_INT_MIN, 0, 400),
```

Comment:
This is specifically restricting the connection interval to 30ms so that is a multiple of both 7.5ms and 10ms SDU interval, which generally provides better chance of scheduling the CIS without conflicts with the ACL :)